projects/step-frontend/src/lib/modules/admin/components/screen-configuration-list/screen-configuration-list.component.ts

```diff
@@ -0,0 +1,113 @@
+import { AfterViewInit, Component, ElementRef, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { downgradeComponent, getAngularJSGlobal } from '@angular/upgrade/static';
+import {
+  AJS_MODULE,
+  ScreenDto,
+  ScreenInputDto,
+  ScreenInputOptionDto,
+  ScreenInputActivationExpressionDto,
+  DialogsService,
+  AuthService,
+  a1Promise2Observable,
+  ContextService,
+  Mutable,
+} from '@exense/step-core';
+import { BehaviorSubject, switchMap, of, catchError, noop, shareReplay, tap, map } from 'rxjs';
+import { ScreenApiService } from '../../services/screen-api.service';
+import { ScreenDialogsService } from '../../services/screen-dialogs.service';
+
+type InProgress = Mutable<Pick<ScreenConfigurationListComponent, 'inProgress'>>;
+
+@Component({
+  selector: 'step-screen-configuration-list',
+  templateUrl: './screen-configuration-list.component.html',
+  styleUrls: ['./screen-configuration-list.component.scss'],
+})
+export class ScreenConfigurationListComponent implements OnDestroy {
+  public readonly CURRENT_SCREEN_CHOICE_DEFAULT = 'executionParameters';
+
+  public currentlySelectedScreenChoice: string = this.CURRENT_SCREEN_CHOICE_DEFAULT;
+
+  readonly _screenChoicesRequest$ = this._screenApi.getScreenChoices();
+  private _screensRequest$ = new BehaviorSubject<unknown>({});
+  readonly screens$ = this._screensRequest$.pipe(
+    tap((_) => ((this as InProgress).inProgress = true)),
+    switchMap((_) =>
+      this._screenApi.getScreenByScreenChoice(this.currentlySelectedScreenChoice).pipe(
+        map((screens: ScreenDto[]) =>
+          screens.map((screen: ScreenDto) => {
+            const screenInput: ScreenInputDto & {
```

Comment:
Returning ScreenInputDto to template and enriching it with new attributes required by the table. This is necessary because the table requires search- and sortable attributes to be frist level.